.github/workflows/markdown.links.config.json

```diff
@@ -3,6 +3,9 @@
     {
       "pattern": "^https://github.com/Lightning-AI/lightning/pull/.*"
     },
+    {
+      "pattern": "^https://pepy.tech/badge/.*"
```

Comment:
This link is often broken. See:
<img width="824" alt="image" src="https://github.com/Lightning-AI/lightning/assets/5495193/52fbaadd-a8d7-4343-a163-b68012bb88d2">

I suggest to remove or replace it with something else

Reply:
ok, I'll check alternatives